docs/v3/package-lock.json

```diff
@@ -1,5768 +1,8 @@
 {
   "name": "api-docs",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I don't think the package-lock.json file should have changed, so please revert this

Reply:
Done